docs/sidebars.js

```diff
@@ -168,6 +168,11 @@ const sidebars = {
           label: 'Use Azure trusted launch VMs',
           id: 'workflows/trusted-launch',
         },
+        {
+          type: 'doc',
+          label: 'Consume Software Bill of Materials (SBOMs)',
```

Comment:
```suggestion
          label: 'Consume SBOMs',
```
This is quite long and produces an ugly line break in the sidebar. Do you think it's okay to shorten it to the acronym?

Reply:
Thanks for noting! I was also unsure during writing, the shorter version is probably better for a consistently styled sidebar.